src/main/java/uk/gov/justice/laa/crime/cfecrime/cma/stubs/utils/CmaResponseUtil.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.justice.laa.crime.cfecrime.cma.stubs.utils;
+
+import lombok.experimental.UtilityClass;
+import lombok.extern.slf4j.Slf4j;
+import uk.gov.justice.laa.crime.cfecrime.Exceptions.UndefinedOutcomeException;
+import uk.gov.justice.laa.crime.cfecrime.api.CfeCrimeRequest;
+import uk.gov.justice.laa.crime.cfecrime.api.CfeCrimeResponse;
+import uk.gov.justice.laa.crime.cfecrime.api.stateless.StatelessApiRequest;
+import uk.gov.justice.laa.crime.cfecrime.api.stateless.StatelessApiResponse;
+import uk.gov.justice.laa.crime.cfecrime.cma.stubs.LocalCmaService;
+import uk.gov.justice.laa.crime.cfecrime.enums.Outcome;
+import uk.gov.justice.laa.crime.cfecrime.utils.FullMeansTestOutcomeCalculator;
+import uk.gov.justice.laa.crime.cfecrime.utils.InitMeansTestOutcomeCalculator;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.CaseType;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.FullAssessmentResult;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.InitAssessmentResult;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.MagCourtOutcome;
+
+@Slf4j
+@UtilityClass
+public class CmaResponseUtil {
+
+
```

Comment:
see comment above about setters. Think we can use constructors for this initialisation

Reply:
I am thinking to leave as is. Later may nee to add more functions for testing.